doc/api/child_process.md

```diff
@@ -147,7 +147,14 @@ added: v0.1.90
 * Returns: {ChildProcess}
 
 Spawns a shell then executes the `command` within that shell, buffering any
-generated output.
+generated output. The `command` string passed to the exec function is processed
+directly by the shell and special characters (vary based on 
+[shell](https://en.wikipedia.org/wiki/List_of_command-line_interpreters))
+need to be dealt with accordingly:
+```js
+exec('"/path/to/test file/test.sh" arg1 arg2'); 
```

Comment:
@thefourtheye - I'm sorry, can't believe I didn't see this. I'm not quite sure I'm following what you're saying. Can you explain a little bit more about what you mean by ? Thanks!

> It might be a good idea to explain the space thing in a comment.